Sources/StringsLintFramework/Rules/Lint/JSONCommentRule.swift

```diff
@@ -198,11 +212,13 @@ extension JSONCommentRule {
         let descriptionString: String?
         let placeholders: [String]?
         let img: String?
+        let max_character_count: Int?
```

Comment:
We try and use snakeCase for variable names.
```suggestion
        let maxCharacterCount: Int?
```

Reply:
I saw Sebastian's using that on Android, so we don't have to match exactly?

Reply:
we don't have to match, no

Reply:
we can have the JSON in the comment be snake_case but the variable we use in the swift linter code should be camelCase